src/twisted/trial/reporter.py

```diff
@@ -96,6 +96,11 @@ class TestResult(pyunit.TestResult):
     expectedFailures: List[Tuple[itrial.ITestCase, str, "Todo"]]  # type: ignore[assignment]
     unexpectedSuccesses: List[Tuple[itrial.ITestCase, str]]  # type: ignore[assignment]
     successes: int
+    # The time when the test was started.
+    # It might be 0 if the tests was skipped, so never started.
+    _testStarted: int
```

Comment:
A `Maybe int` (so, possibly `Optional[int]`) would be a better way to represent "time when test started, or never started" would.  Leaving 0 here lets code easily incorrectly interpret the start time as the UNIX epoch.

Likewise for `_lastTime`.